### ISSUE TITLE
Missing "es" and "lib" directories in NPM published v2.11.0

### ISSUE THREAD

**Issue by Menci** [reactchartjs/react-chartjs-2#588]
```json
{
  "name": "react-chartjs-2",
  "version": "2.11.0",
  "description": "react-chartjs-2",
  "main": "lib/index.js",
  "module": "es/index.js",
  "jsnext:main": "es/index.js",
  "author": "reactchartjs"
}
```

```bash
$ find .
.
./.circleci
./.circleci/config.yml
./stories
./stories/MixLineBar.js
./stories/StockExamples.js
./stories/UpdatingChart.js
./stories/Welcome.js
./stories/index.js
./test
./test/config
./test/config/setup.js
./test/__tests__
./test/__tests__/Chart_spec.js
./LICENSE.md
./README.md
./index.d.ts
./package.json
```

**Comment by vitdolinek** [comment]
Hello, I can confirm this issue for versions 2.9.0 and 2.10.0 as well.

**Comment by argshook** [comment]
> Hello, I can confirm this issue for versions 2.9.0 and 2.10.0 as well.

hey, are you sure about `2.10.0`? I see `lib/index.js` and `es/index.js` in that version:

```sh
$ npm pack react-chartjs-2@2.10.0
npm notice
npm notice 📦  react-chartjs-2@2.10.0
npm notice === Tarball Contents ===
npm notice 8.3kB   test/__tests__/Chart_spec.js
npm notice 22.8kB  es/index.js
npm notice 22.8kB  lib/index.js
npm notice 186B    stories/index.js
npm notice 4.5kB   stories/MixLineBar.js
npm notice 153.7kB dist/react-chartjs-2.js
npm notice 28.6kB  dist/react-chartjs-2.min.js
npm notice 547B    test/config/setup.js
npm notice 1.6kB   stories/StockExamples.js
npm notice 3.0kB   stories/UpdatingChart.js
npm notice 1.1kB   stories/Welcome.js
npm notice 3.4kB   package.json
npm notice 7.4kB   README.md
npm notice 2.0kB   index.d.ts
npm notice 301B    .circleci/config.yml
```

but they're gone in 2.11.0:

```sh
$ npm pack react-chartjs-2@2.11.0
npm notice
npm notice 📦  react-chartjs-2@2.11.0
npm notice === Tarball Contents ===
npm notice 8.3kB test/__tests__/Chart_spec.js
npm notice 186B  stories/index.js
npm notice 4.5kB stories/MixLineBar.js
npm notice 547B  test/config/setup.js
npm notice 1.6kB stories/StockExamples.js
npm notice 3.0kB stories/UpdatingChart.js
npm notice 1.1kB stories/Welcome.js
npm notice 3.4kB package.json
npm notice 1.1kB LICENSE.md
npm notice 7.4kB README.md
npm notice 2.0kB index.d.ts
npm notice 301B  .circleci/config.yml
...
```

**Comment by argshook** [comment]
summoning @jerairrest :)

**Comment by jerairrest** [comment]
Sorry about that! These should be published in 2.11.1.
